WMF Framework/Wikipedia.xcdatamodeld/Wikipedia.xcdatamodel/contents

```diff
@@ -79,6 +79,7 @@
         <attribute name="imageWidth" optional="YES" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="NO" syncable="YES"/>
         <attribute name="isDownloaded" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES" syncable="YES"/>
         <attribute name="isExcludedFromFeed" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES" syncable="YES"/>
+        <attribute name="isSavedToDisk" attributeType="Boolean" defaultValueString="YES" usesScalarValueType="YES" syncable="YES"/>
```

Comment:
Seems there may be overlap between `isSavedToDisk` and `isDownloaded`. How do they differ?

Reply:
`isSavedToDisk` indicates if the save operations were successful. `isDownloaded` indicates if fetch was successful. `isDownloaded` will be set to `YES`, even if save to disk operations fail. If we want to modify the code to rely on `isDownloaded`, we'd still need something that tell us if the article is queued to be downloaded or if save to disk operations failed

Reply:
I see. Formerly I think I may have assumed `isDownloaded` was not set to true if the data could not be saved.

Reply:
If you comment out https://github.com/wikimedia/wikipedia-ios/blob/bf5086e63d9f7ae91084c2870c9540d247c2b591/Wikipedia/Code/MWKDataStore.m#L1127 and then try to save an article, its sections won't be saved but `isDownloaded` will be set to `YES`

Reply:
That's a bug and these errors should bubble up to the `SavedArticlesFetcher`. Looking at addressing this in the other PR